dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -517,22 +491,24 @@ export const DynamicFast = ({
 							) : (
 								// Regular standards layout
 								standards.map((card, cardIndex) => {
+									const { columns } = standardsDisplayConfig;
+									const fullRows = Math.floor(
+										standards.length / columns,
+									);
 									return (
 										<LI
 											key={card.url}
 											percentage={
 												standardsDisplayConfig.cardWidth
 											}
 											stretch={true}
-											showDivider={true}
+											showDivider={
+												cardIndex % columns !== 0
+											}
```

Comment:
This is a drive by fix to a divider bug I presume? Just checking there's not a connection to the gap I'm missing